doc/api/globals.markdown

```diff
@@ -162,6 +166,21 @@ left untouched.
 Use the internal `require()` machinery to look up the location of a module,
 but rather than loading the module, just return the resolved filename.
 
+## setImmediate(callback[, arg][, ...])
+
+To schedule the "immediate" execution of `callback` after I/O events'
```

Comment:
This is copied verbatum from the timers documentation. If changed in one place should change in another. That alright?